db/migrate/20190705114625_add_https_status_to_podcast_episodes.rb

```diff
@@ -0,0 +1,7 @@
+class AddHttpsStatusToPodcastEpisodes < ActiveRecord::Migration[5.2]
+  def change
+    add_column :podcast_episodes, :https, :boolean, default: true
+
+    PodcastEpisode.where("media_url like ?", "http:/%").update_all(https: false)
```

Comment:
Although this is probably safe, it could be a problem if the model is ever renamed, take a look at the example "Redefine models in migrations" in https://railsguides.net/change-data-in-migrations-like-a-boss/

Also, you should probably use `ILIKE`, for the same reason I mentioned earlier, http might not be lowercase